assembl/models/attachment.py

```diff
@@ -371,3 +371,40 @@ class ResourceAttachment(Attachment):
     crud_permissions = CrudPermissions(
         P_MANAGE_RESOURCE, P_READ, P_MANAGE_RESOURCE, P_MANAGE_RESOURCE,
         P_MANAGE_RESOURCE, P_MANAGE_RESOURCE)
+
+
+class AgentProfileAttachment(Attachment):
+
+    __tablename__ = "agent_profile_attachment"
+
+    id = Column(Integer, ForeignKey(
+        'attachment.id',
+        ondelete='CASCADE',
+        onupdate='CASCADE'
+    ), primary_key=True)
+
+    user_id = Column(Integer, ForeignKey(
+        'agent_profile.id',
+        ondelete='CASCADE',
+        onupdate='CASCADE',
+        ),
+        nullable=False,
+        index=True)
+
+    user = relationship(
+        AgentProfile,
+        backref=backref(
+            'profile_attachments',
+            cascade="all, delete-orphan"),
+    )
+    __mapper_args__ = {
+        'polymorphic_identity': 'agent_profile_attachment',
+        'with_polymorphic': '*'
+    }
+
+    crud_permissions = CrudPermissions(
+            P_READ, P_SYSADMIN, P_SYSADMIN, P_SYSADMIN,
```

Comment:
Particiapnts can Create, Sysadmin can RUD? Is that right?

Reply:
yes, that's right and only owner can update, remove